From 6ffd9db8e02e411bda2f421abf7951dab6cf0e38 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 5 Feb 2021 01:43:44 -0800
Subject: [PATCH] configure.ac: explicitly link to jpeg lib

Since oe has added lib jpeg to the depends, explicitly link to jpeg lib
to workaround build failure with autoconf 2.7

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1ce81f91..8bbb50de 100644
--- a/configure.ac
+++ b/configure.ac
@@ -191,24 +191,12 @@ AC_SUBST(SSL_CFLAGS)
 AC_SUBST(SSL_LIBS)
 AS_VAR_APPEND([SPICE_REQUIRES], [" openssl"])
 
-AC_CHECK_LIB(jpeg, jpeg_destroy_decompress,
-    AC_MSG_CHECKING([for jpeglib.h])
-    AC_TRY_CPP(
-[#include <stdio.h>
-#undef PACKAGE
-#undef VERSION
-#undef HAVE_STDLIB_H
-#include <jpeglib.h>],
-        JPEG_LIBS='-ljpeg'
-        AC_MSG_RESULT($jpeg_ok),
-	AC_MSG_ERROR([jpeglib.h not found])),
-    AC_MSG_ERROR([libjpeg not found]))
+JPEG_LIBS='-ljpeg'
 AC_SUBST(JPEG_LIBS)
 
 AC_CHECK_LIB(z, deflate, Z_LIBS='-lz', AC_MSG_ERROR([zlib not found]))
 AC_SUBST(Z_LIBS)
 
-
 AC_ARG_ENABLE([manual],
                AS_HELP_STRING([--enable-manual=@<:@auto/yes/no@:>@],
                               [Build SPICE manual]),
-- 
2.29.2

